### DISCUSSION TITLE
How long will the incomplete object be deleted?

### DISCUSSION THREAD

**Discussion by rhypowang** [minio/minio#12744] [Q&A] [answered]

**Comment by harshavardhana** [comment, marked as answer]
24 hours or older.

**Reply by DzianisMarkhotka** [reply]
Is it configurable? Are you support AbortIncompleteMultipartUpload as well as S3?

**Reply by harshavardhana** [reply]
No it is not configurable @DzianisMarkhotka

**Reply by klauspost** [reply]
Yes, `AbortIncompleteMultipartUpload` will delete the data.

**Reply by T-TRz879** [reply]
Sorry, I have a new question about my answer from 3 years ago. I would like to know the reason why AbortIncompleteMultipartUpload is not supported. Is it because there is a lot of data that is not merged and users do not configure cleaning tasks

**Reply by harshavardhana** [reply]
It is supported, you just call the API with the upload id that you want to abort.
